client/my-sites/people/people-list-item/index.jsx

```diff
@@ -42,29 +45,41 @@ export default React.createClass( {
 			this.userHasPromoteCapability() &&
 			! this.props.isSelectable
 		);
-	},
+	};
 
 	render() {
 		const canLinkToProfile = this.canLinkToProfile();
+
 		return (
 			<CompactCard
-				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove' ) }
+				type={ this.props.type }
 				className={ classNames( 'people-list-item', this.props.className ) }
 				tagName="a"
 				href={ canLinkToProfile && '/people/edit/' + this.props.site.slug + '/' + this.props.user.login }
-				onClick={ canLinkToProfile && this.navigateToUser }>
+				onClick={ canLinkToProfile && this.props.navigateToUser }>
 				<div className="people-list-item__profile-container">
 					<PeopleProfile user={ this.props.user } />
 				</div>
 				{
 				this.props.onRemove &&
 				<div className="people-list-item__actions">
 					<button className="button is-link people-list-item__remove-button" onClick={ this.props.onRemove }>
-						{ this.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
+						{ this.props.translate( 'Remove' ) }
 					</button>
 				</div>
 				}
 			</CompactCard>
 		);
 	}
-} );
+}
+
+const navigateToUser = () => {
+	browserWindow.scrollTo( 0, 0 );
```

Comment:
@markryall this is a good move. I think it may be worth considering one more step in this process here before we merge. I don't quite understand what the reason is for scrolling here and I think that comes into play. It doesn't appear to be the result of a navigation.

Whatever it is, it's probably describable by some Redux action, even if it requires no state in the Redux tree. In that case I could imagine simply dispatching the action and using some watcher to make sure we scroll. That would entirely eliminate the `browser` object from this model.

```js
const mapDispatchToProps = ( {
	navigateToUser: withAnalytics(
		recordGoogleEvent( 'People', 'Clicked User profile from team list' ),
		{ type: PEOPLE_USER_PROFILE_SELECT, userId: this.props.user.ID },
	),
} );

// in some new scrolling middleware
const inBrowser = 'undefined' !== typeof window;

const scrollToTop = () => window.scrollTo( 0, 0 );

export const handlers =
	inBrowser
		? { [ PEOPLE_USER_PROFILE_SELECT ]: [ scrollToTop ] }
		: {};

export default handlers;

// state/index.js
const enhancers = [
	…
	inBrowser && scrollHandlers
	…
];
```